README.md

```diff
@@ -22,20 +22,19 @@ Follow the following steps to install the BrickPi_Python Module on a Raspbian Je
 4. Run `sudo git clone http://www.github.com/DexterInd/BrickPi.git`
 5. Run `cd BrickPi/Setup\ Files`
 6. Run `sudo bash install.sh`
-7. Run `sudo pip install -U future`
-8. `cd ..`
-9. Run `sudo git clone http://www.github.com/DexterInd/BrickPi_Python.git`
-10. Run `sudo apt-get install python-setuptools python-dev build-essential`
-11. Run `cd BrickPi_Python`
-12. Run `sudo python setup.py install` to install the `BrickPi` module globally.
-13. Run `sudo raspi-config` and go to "7 Advanced Options" --> "A8 Serial" --> And then select "No"
-14. Modify /boot/config.txt to enable_uart=1 (This line should be there from the previous install, but set to 0 from the raspi-config commands
+7. `cd ..`
+8. Run `sudo git clone http://www.github.com/DexterInd/BrickPi_Python.git`
```

Comment:
Hey @Shoban94 a lot of these things are already taken care of since this code now resides in BrickPi.  So cloning BrickPi_Python for example is redundant.